### ISSUE TITLE
check-mysql-disk.rb error : CheckMysqlDisk CRITICAL: comparison of Float with String failed

### ISSUE THREAD

**Issue by taejoonmoon** [sensu-plugins/sensu-plugins-mysql#66]
I'm not familiar with ruby.

When I run check-mysql-disk.rb, I found this error.
"CheckMysqlDisk CRITICAL: comparison of Float with String failed"
```
$ check-mysql-disk.rb -h 127.0.0.1 -i /etc/sensu/conf.d/mysql.cnf --size 1000
CheckMysqlDisk CRITICAL: comparison of Float with String failed
```

If I add "to_f" to disk_size, critical_usage, warning_usage, it works well.
Is this the right way? If then, I will make a pull request.

```
$ diff check-mysql-disk.rb test-check-mysql-disk.rb
88,90c88,90
<     disk_size = config[:size]
<     critical_usage = config[:crit]
<     warning_usage = config[:warn]
---
>     disk_size = config[:size].to_f
>     critical_usage = config[:crit].to_f
>     warning_usage = config[:warn].to_f
```

**Comment by majormoses** [comment]
Hey sorry for the late response, let me review the code real quick to verify but that looks fairly reasonable to me.

**Comment by majormoses** [comment]
So that works but I think that it would be better to fix the defaults to not be a string in the first place. We see in multiple locations we know we want to convert it to a float but then we set the default to a string which makes no sense. My conclusion is that the proc block does not run on the default value.
- https://github.com/sensu-plugins/sensu-plugins-mysql/blob/2.2.0/bin/check-mysql-disk.rb#L54-L55
- https://github.com/sensu-plugins/sensu-plugins-mysql/blob/2.2.0/bin/check-mysql-disk.rb#L61-L62

Your fix is totally valid but I think we should remove the quotes from the default and confirm that it works with default and non default values. I look forward to reviewing and getting this fixed.

**Comment by jtomaszon** [comment]
That's right, putting real number instead of string on default value did the trick
